pydantic/fields.py

```diff
@@ -663,6 +673,7 @@ def Field(  # noqa: C901
         examples: Example values for this field.
         exclude: Whether to exclude the field from the model serialization.
         discriminator: Field name or Discriminator for discriminating the type in a tagged union.
+        deprecated: A deprecation message to be emitted when accessing the field.
```

Comment:
Similarly, we could update the docstrings to reflect the fact that you can pass a `str` or `deprecated` instance :)